packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -276,12 +281,20 @@ export class LustreFileSystem extends FileSystemBase {
   private validatePerUnitStorageThroughput(deploymentType: LustreDeploymentType, perUnitStorageThroughput?: number) {
     if (perUnitStorageThroughput === undefined) { return; }
 
-    if (deploymentType !== LustreDeploymentType.PERSISTENT_1) {
-      throw new Error('perUnitStorageThroughput can only be set for the PERSISTENT_1 deployment type');
+    if (deploymentType !== LustreDeploymentType.PERSISTENT_1 && deploymentType !== LustreDeploymentType.PERSISTENT_2) {
+      throw new Error('perUnitStorageThroughput can only be set for the PERSISTENT_1/PERSISTENT_2 deployment types');
```

Comment:
Let's add what was provided here (this is a good practice with all error messages):

```suggestion
      throw new Error('perUnitStorageThroughput can only be set for the PERSISTENT_1/PERSISTENT_2 deployment types, received: ' + deploymentType);
```

Reply:
Agreed. Updated as part of Revision 4